Martinh/update wtt demos (#614)
wormhole-foundation/wormhole-docs: products/token-transfers/wrapped-token-transfers/tutorials/transfer-workflow.md

    echo ".env" >> .gitignore
    ```

3. **Install dependencies**: Install the required dependencies.

    ```bash

>     npm install @wormhole-foundation/sdk dotenv tsx

Reviewer: We still have devs installing dotenv in demos? Why?

Author: Good catch! we shouldn't

    ```

> 4. **Set up environment variables**: To securely store your private key, create a `.env` file in the root of your project.

Reviewer: This needs updated to reflect the newer language around managing wallets and keys, please.

    --8<-- "code/products/wrapped-token-transfers/tutorials/transfer-workflow/wtt-3.ts:30:30"
    ```

> 8. **Set transfer mode**: Specify that the transfer should be manual by setting `route = 'TokenBridge'`. This means you will need to handle the attestation and finalization steps yourself. To use automatic relaying on EVM chains, set `route = 'AutomaticTokenBridge'`.

Reviewer: ```suggestion
8. **Set transfer mode**: Specify manual or automatic transfer using `route`. Set `route  = 'TokenBridge'` for manual transfers, where you will handle the attestation and finalization steps yourself. To use automatic relaying on EVM chains, set `route = 'AutomaticTokenBridge'`.
```
